kiva/gl/__init__.py

```diff
@@ -442,8 +446,11 @@ def GetLabel(text, pyglet_font):
         return label
 
 
-except ImportError:
+except ImportError as exc:
     # Pyglet is not available, so we forgo some features
+    logger.info(
```

Comment:
Why not `logger.exception`?

Reply:
It felt more natural to go from quietly carrying with no warning to giving an info worning which might be useful, vs a logged exception.

Possibly a good idea to at least include the exception info, though, since otherwise it may be hard to workout what is missing.

Reply:
Sounds good